app/profile/index.js

```diff
@@ -21,12 +38,17 @@ export default class LoginScreen extends React.Component<{||}> {
 
   render = () => {
     return (
-      <LayoutSingleColumn>
-        <SimpleCard>
-          <Login onLogin={this.onLogin} />
-          <Logout onLogout={this.onLogout} />
-        </SimpleCard>
-      </LayoutSingleColumn>
+      <DismissKeyboardView>
+        <LayoutSingleColumn>
+          <SimpleCard>
+            <Login onLogin={this.onLogin} />
+            <Logout onLogout={this.onLogout} />
+          </SimpleCard>
+          {this.state.isLoginSuccess && (
+            <Translation passThrough="Successfully logged in" />
+          )}
```

Comment:
What about displaying login for if logged in and logout button otherwise? It would make more sense to me...